src/graphql/typeDefs/mutations/party/partyUpdate.ts

```diff
@@ -14,6 +14,8 @@ export default gql`
 		name: String
 		location: String
 		radius: Int
+		price: [Int]
```

Comment:
```suggestion
		price: [Int!]
```

this needs to be none-nullable because values like `[null]` could be passed in